src/docs/asciidoc/core/core-beans.adoc

```diff
@@ -6659,7 +6659,7 @@ annotated classes. The following example shows how to do so:
 [source,kotlin,indent=0,subs="verbatim,quotes",role="secondary"]
 .Kotlin
 ----
-	@Component
+	@Configuration
```

Comment:
The documentation is correct.
We can define methods with **@Bean** annotation inside any **@Component** classes.